src/commentCounter.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @jest-environment jsdom
+ */
+import counter from './commentCounter.js';
+
+describe('testing the comment counting function', () => {
+  test('the comment container returns count of 1 after appending comment element', () => {
+    const container = document.createElement('div');
+    const comment = document.createElement('p');
+    comment.innerText = 'Easy peasy lemon squeazy';
+    container.appendChild(comment);
+    expect(counter(container)).toBe(1);
+  });
+});
```

Comment:
- [ Required ] Good job adding tests for the `commentCounter` module, however, I think it would be better if you make sure that you have tests for all edge cases e.g think at least about the scenario in which there are 0 items on the page (counter function should return 0)

## Example code
```JavaScript
describe('testing the comment counting function', () => {
  test('the comment container returns count of 0', () => {
    const container = document.createElement('div');
    const comment = document.createElement('p');
    expect(counter(container)).toBe(0);
  });
});
```